backend/app/models/user.rb

```diff
@@ -13,10 +13,11 @@ class User < ApplicationRecord
   acts_as_follower
   acts_as_followable
 
-  validates :username, uniqueness: { case_sensitive: true },
-                       format: { with: /\A[a-zA-Z0-9]+\z/ },
-                       presence: true,
-                       allow_blank: false
+  validates :username,
+    uniqueness: { case_sensitive: true, message: 'not available' },
+    format: { with: /\A[._a-zA-Z0-9]+\z/, message: ->(user, data) { "only allows letters and digits. Got #{data[:value]}" } },
```

Comment:
Changed because the username format is unnecessarily restrictive and would only annoy users